addons/website/static/tests/tours/edit_menus.js

```diff
@@ -356,6 +356,106 @@ wTourUtils.registerWebsitePreviewTour('edit_menus', {
         trigger: ':iframe .top_menu .nav-item:contains("Home") li:contains("Contact us")',
         run: "press ArrowDown",
     },
+    ...wTourUtils.clickOnSave(),
+    // Nest and re-arrange menu items for a newly created menu
+    {
+        content: "Open site menu",
+        trigger: 'button[data-menu-xmlid="website.menu_site"]',
+        run: "click",
+    },
+    {
+        content: "Click on Edit Menu",
+        trigger: 'a[data-menu-xmlid="website.menu_edit_menu"]',
+        run: "click",
+    },
+    {
+        content: "Trigger link dialog (click 'Add Menu Item')",
+        trigger: '.modal-body a:eq(0)',
+        run: "click",
+    },
+    {
+        content: "Write a label for the new menu item",
+        trigger: '.modal-dialog .o_website_dialog input:eq(0)',
+        run: "edit new_menu",
+    },
+    {
+        content: "Write a url for the new menu item",
+        trigger: '.modal-dialog .o_website_dialog input:eq(1)',
+        run: "edit #",
+    },
+    {
+        content: "Confirm the new menu entry",
+        trigger: '.modal-footer .btn-primary',
+        run: "click",
+    },
+    {
+        content: "Check if new menu(new_menu) is added",
+        trigger: '.oe_menu_editor li:contains("new_menu")',
+        run: () => {}, // It's a check.
+    },
+    {
+        content: "Trigger link dialog (click 'Add Menu Item')",
+        trigger: '.modal-body a:eq(0)',
+        run: "click",
+    },
+    {
+        content: "Write a label for the new menu item",
+        trigger: '.modal-dialog .o_website_dialog input:eq(0)',
+        run: "edit new_nested_menu",
+    },
+    {
+        content: "Write a url for the new menu item",
+        trigger: '.modal-dialog .o_website_dialog input:eq(1)',
+        run: "edit #",
+    },
+    {
+        content: "Confirm the new menu entry",
+        trigger: '.modal-footer .btn-primary',
+        run: "click",
+    },
+    {
+        content: "Check if new menu(new_nested_menu) is added",
+        trigger: '.oe_menu_editor li:contains("new_nested_menu")',
+        run: () => {}, // It's a check.
```

Comment:
this is not needed... Doing nothing has become the “default” action (instead of “clicking”)

Reply:
Thanks...I'll make the suggested changes 🙂